styleguide.config.js

```diff
@@ -45,45 +47,57 @@ module.exports = {
                 {
                     test: /\.js?$/,
                     exclude: /node_modules/,
-                    loader: 'babel-loader'
+                    loader: 'babel-loader',
                 },
                 {
                     test: /\.css$/,
-                    use: [
+                    loader: [
                         'style-loader',
-                        'css-loader'
-                    ]
-                }
-            ]
-        }
+                        'css-loader',
+                    ],
+                },
+            ],
+        },
     },
 
-    styleguideDir: 'docs'
-};
+    sections: [
+        {
+            name: 'Introduction',
+            content: './README.md',
+        },
+        {
+            name: 'Charts',
+            sections: [
+                {
+                    name: 'Stacked Area Chart',
+                    components: 'src/charts/stackedArea/StackedAreaComponent.js',
+                },
+                {
+                    name: 'Donut Chart',
+                    components: 'src/charts/donut/DonutComponent.js',
+                },
+            ],
+        },
+        {
+            name: 'Chart Components',
+            sections: [
+                {
+                    name: 'Legend',
+                    components: 'src/charts/legend/LegendComponent.js',
+                },
+                {
+                    name: 'Tooltip',
+                    components: 'src/charts/tooltip/TooltipComponent.js',
+                },
+            ],
+        },
+
+        // {
+        //     name: 'Changelog',
+        //     content: './CHANGELOG.md',
+        // },
```

Comment:
Can this commented-out code be removed as well?

Reply:
Good call!